src/evolution/mp_evolutionary_strategy.py

```diff
@@ -176,12 +176,9 @@ def __exit__(self, *args) -> None:
         self.pool.terminate()
 
     @staticmethod
```

Comment:
what is this? comments please?

Reply:
I am not going to bother commenting the multiprocessing stuff because it is going to get removed as soon as I finish the MPI stuff. There is a bug that is fundamentally not fixable with python, and multiprocessing sucks in general lol.